sdk/storage/test/test_base.h

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#pragma once
+
+#include "gtest/gtest.h"
+
+namespace Azure
+{
+namespace Storage
+{
+namespace Test
+{
+  struct TestUtility 
```

Comment:
Is this struct necessary?

Reply:
I don't like the idea of a global variable within a namespace that everyone can access, except for the variables that will be used outside of the library. This is to prevent that.

Reply:
You didn't prevent anything, they can still access it  with `Azure::Storage::Test::TestUtility::xxx`

Reply:
Plus I don't think connection string is any kind of **utility**.

Reply:
@katmsft why is this marked as resolved?